airbyte-integrations/connectors/source-newsdata/source_newsdata/newsdata.yaml

```diff
@@ -0,0 +1,80 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["results"]
+  base_requester:
+    url_base: "https://newsdata.io/api/1"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "X-ACCESS-KEY"
+      api_token: "{{ config['api_key'] }}"
```

Comment:
From the API documentation:
> To obtain an API key you have to [create an account](https://newsdata.io/register). You get a free API key while you are in development. When making REST API requests, your API key has to be provided as a parameter in the URL (e.g. apikey=YOUR_API_KEY).

I tried to read data but no records were read.
```json
{"type": "LOG", "log": {"level": "INFO", "message": "Read 0 records from latest stream"}}
```

Reply:
Please check the authentication.

Reply:
If the request would not have been authenticated, you would have gotten a 401 - Unauthorized response.

From Newsdata API:

> Your Newsdata.io API Key. Alternatively you can provide this via the X-ACCESS-KEY HTTP header.

Could you share your `config.json`?